src/track/track_gps_l2cm.c

```diff
@@ -432,9 +432,20 @@ static void tracker_gps_l2cm_update(const tracker_channel_info_t *channel_info,
   corr_t* cs = data->cs;
 
   /* Update C/N0 estimate */
+
+  /* Pre-computed C/N0 estimator and filter parameters. The parameters are
+   * computed using equivalent of cn0_est_compute_params() function for
+   * integration period of 20ms and cut-off frequency of 0.1 Hz.
+   */
+  static const cn0_est_params_t pre_computed = {
+    1.6989700e+01f, 3.9130205e-05f, -1.9822289e+00f, 9.8238545e-01f
```

Comment:
Based on modifications to https://github.com/swift-nav/piksi_firmware/pull/723
the improved coefficients are:
1.6989700e+01f, 3.9129183e-05f, -1.9822294e+00f, 9.8238576e-01f